pkg/result/result.go

```diff
@@ -10,8 +10,9 @@ type DynamicAnalysisStraceSummary map[pkgecosystem.RunPhase]*StraceSummary
 type DynamicAnalysisFileWrites map[pkgecosystem.RunPhase]*FileWrites
 
 type DynamicAnalysisResults struct {
-	StraceSummary DynamicAnalysisStraceSummary
-	FileWrites    DynamicAnalysisFileWrites
+	StraceSummary    DynamicAnalysisStraceSummary
+	FileWrites       DynamicAnalysisFileWrites
+	FileWriteBuffers []string
```

Comment:
Given this struct is written to .json file, is this field populated in the final result .json, as well as the bucket?

Reply:
This fields in this struct are written in separate write calls. (If you take a look at worker/main.go and analyze/main.go) so the FileWriteBuffers struct is not written to any .json file. StraceSummary and FileWrites are also written to separate .json files.